src/penn_chime/parameters.py

```diff
@@ -78,4 +99,4 @@ def change_date(self):
         longer match current results, indicating when users should
         re-run their reports
         """
-        return "March 23 2020"
+        return "March 27 2020"
```

Comment:
we should make this the date of deploy

Reply:
Agreed, I think this should go into models.py too though.